src/classes/game.rb

```diff
@@ -0,0 +1,19 @@
+require_relative './item'
```

Comment:
Hi **@Beardless-sheik,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

## optional

- [ ] **You can make your directory clean. There are unneeded files outside the `my-app` folder. I advise you to delete the my-app folder and put its contents in the root directory.**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```